sr_gui_grasp_controller/CMakeLists.txt

```diff
@@ -1,10 +1,10 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(sr_gui_grasp_controller)
-find_package(catkin REQUIRED COMPONENTS rospy rqt_gui rqt_gui_py sr_hand sr_visualization_icons)
+find_package(catkin REQUIRED COMPONENTS rospy rqt_gui rqt_gui_py sr_hand sr_visualization_icons sr_robot_commander)
 
 catkin_package(
-    DEPENDS
-    CATKIN_DEPENDS rospy rqt_gui rqt_gui_py sr_hand sr_visualization_icons
+    DEPENDS sr_robot_commander
```

Comment:
I think the `DEPENDS` is for system dependencies. So you should have it only in `CATKIN_DEPENDS`